pulp/mps_lp.py

```diff
@@ -83,6 +86,12 @@ def readMPS(path, sense, dropConsNames=False):
                     mode = CORE_FILE_BOUNDS_MODE_NAME_GIVEN
                 else:
                     mode = CORE_FILE_BOUNDS_MODE_NO_NAME
+            elif parameters['sense'] is None and line[0] == CORE_FILE_OBJSENSE_MODE:
+                if len(line) > 1:
+                    parameters['sense'] = const.LpMaximize if line[1] == 'MAX'\
+                            else const.LpMinimize
+                else:
+                    mode = CORE_FILE_OBJSENSE_MODE
```

Comment:
I'm not sure why would we have the same mode regardless of the if clause. Maybe we should put the assignment before the `len(line)>1`  and take out the `else` clause? Or maybe I don't understand the idea.

Reply:
Following on from the previous comment, we have to enter the `CORE_FILE_OBJSENSE_MODE` is the objective sense is not given in the same line as the `OBJSENSE` keyword. This is catching the case
```
OBJSENSE
  MAX
```
The check of `len(line) > 1` differentiates between these two cases.